e/erubi/erubi_ubi8.4.sh

```diff
@@ -0,0 +1,64 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: erubi
+# Version	: v1.10.0
+# Source repo	: https://github.com/jeremyevans/erubi
+# Tested on	: UBI 8.4
+# Language      : Ruby
+# Travis-Check  : False
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sapana Khemkar {Sapana.khemkar@ibm.com}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e 
+
+PACKAGE_NAME=erubi
+PACKAGE_VERSION=1.10.0
```

Comment:
can we make this configurable / accept input?

Reply:
done